.changes/unreleased/Features-20220907-154639.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Features
```

Comment:
nit:  This seems like it's `Under the Hood` instead of a `Feature`.

Reply:
Changed.